detection_rules/integrations.py

```diff
@@ -262,8 +283,10 @@ def get_integration_schema_data(data, meta, package_integrations: dict) -> Gener
                 min_stack = meta.min_stack_version or load_current_package_version()
 
                 # Prior to 8.3, some rules had a min_stack_version with only major.minor
-                if Version(min_stack) != 3:
-                    min_stack = Version(Version(load_current_package_version()) + (0,))
+                if len(min_stack.split(".")) < 3:
+                    min_stack = Version(*min_stack.split("."))
```

Comment:
Might not need this now if we pass the optional flag.

```python
>>> semver.Version.parse("8.0", optional_minor_and_patch=True) == semver.Version.parse("8.0.0")
True
>>>
```

Reply:
Agreed, this has been updated already.